src/layouts/Layout.astro

```diff
@@ -130,4 +134,11 @@ const { title } = Astro.props;
 		white-space: nowrap;
 		border-width: 0;
 	}
+
+	a:focus-visible {
+		outline-color: #FF4040;
+		outline-width: 2px;
+		outline-offset: 2px;
```

Comment:
Adding an `outline-style: solid;` seemed to break focus styles in the grid 🤔 
May just be my very outdated browser?

Reply:
This is actually a bug in CSS. It's because the link is an `inline` element with `block` level children. Here is the [demo](https://codepen.io/davidleininger/pen/XWQLxZy/8ec4fcf01235a1a22e7ee2e7b5f17a64) I made for the CSSWG to look at to figure out the issue. I've updated the rest of the code to get solid working.

Reply:
ah, nice! TIL! thanks!